backend/src/entity/projects.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn } from 'typeorm';
+
+@Entity('project')
+export class Project {
+    @PrimaryGeneratedColumn({ type: 'bigint' })
+    id: number;
+
+    @Column({ type: 'varchar', length: 255, nullable: false })
+    name: string;
+
+    @Column({ type: 'text', nullable: false })
+    description: string;
+
+    @Column({ type: 'varchar' })
+    image: string;
+
+    @CreateDateColumn({ type: 'timestamptz', default: () => 'CURRENT_TIMESTAMP' })
+    created_at: Date;
+
+    @Column({ type: 'bigint', nullable: false })
+    leader_id: number;
+
+    @Column({ type: 'bigint', array: true, default: '{}' })
+    members_id: number[];
```

Comment:
Are you sure default array is '{}' ?

Reply:
It creates me an error, during the connection to the db, if i put '[]'